<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>repairManagement</class>
 <widget class="repairManagement" name="repairManagement">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>repairManagement</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QStackedWidget" name="stackedWidget">
     <widget class="QWidget" name="pageOfRaiseRepair">
      <layout class="QVBoxLayout" name="verticalLayout_3" stretch="1,0,2">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="font">
          <font>
           <family>Arial</family>
           <pointsize>15</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>申请报修</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTextEdit" name="contentOfRaise"/>
       </item>
       <item>
        <widget class="QPushButton" name="btnOfRaise">
         <property name="font">
          <font>
           <family>Arial</family>
           <pointsize>15</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>提交</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="pageOfHandleRepair">
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="font">
          <font>
           <family>Arial</family>
           <pointsize>15</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>处理报修</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTableWidget" name="recordOfHandle"/>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QPushButton" name="btnOfRefreshForHandle">
           <property name="font">
            <font>
             <family>Arial</family>
             <pointsize>15</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>刷新</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnOfHandle">
           <property name="font">
            <font>
             <family>Arial</family>
             <pointsize>15</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>处理</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="pageOfSearchRepair">
      <layout class="QVBoxLayout" name="verticalLayout_2" stretch="1,0,2">
       <item>
        <widget class="QLabel" name="label">
         <property name="font">
          <font>
           <family>Arial</family>
           <pointsize>15</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>查询报修</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTableWidget" name="recordOfRepair"/>
       </item>
       <item>
        <widget class="QPushButton" name="btnOfRefresh">
         <property name="font">
          <font>
           <family>Arial</family>
           <pointsize>15</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>刷新</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>repairManagement</class>
   <extends>QWidget</extends>
   <header>repairmanagement.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
